CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 7.0.0
+
+- Improved version of `analysis_options`
```

Comment:
Keeping a change log is not quite useful if you're not going to log the actual changes.